### PR TITLE
Feat: Fix content page overview page

### PR DESCRIPTION
**Pull request by bertyhell**
image
image

### CODE REVIEW

File: demo/src/react-admin/modules/content-page/queries/hetarchief/get-content-by-id.graphql [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
+            id
+            position
+            updated_at
+            variables
+            content_block_type
+        }
     }
-  }
 }
+# TODO add the information fetch back to maintainer once org api is back up and running
```
Comment (bartnaessens) [inline demo/src/react-admin/modules/content-page/queries/hetarchief/get-content-by-id.graphql:55]: Check: seems to be back for local/int?

File: demo/src/react-admin/modules/content-page/queries/hetarchief/get-content-pages.graphql [hunk 2]
```diff
@@ -1,53 +1,54 @@
...
-    aggregate {
-      count
+    app_content_page_aggregate(where: $where) {
+        aggregate {
+            count
+        }
     }
-  }
 }
+# TODO add the information fetch back to maintainer once org api is back up and running
```
Comment (bartnaessens) [inline demo/src/react-admin/modules/content-page/queries/hetarchief/get-content-pages.graphql:49]: check

File: demo/src/react-admin/modules/content-page/services/content-page.service.ts [hunk 3]
```diff
@@ -608,25 +604,25 @@ export class ContentPageService {
...
-				filters,
-				page,
-				size,
-				orderProp,
-				orderDirection,
-			},
-		});
-		return parsed;
-	}
+	// public static async getContentPages(
```
Comment (bartnaessens) [inline demo/src/react-admin/modules/content-page/services/content-page.service.ts:607]: correct that this is commented?

Comment (bertyhell) [reply demo/src/react-admin/modules/content-page/services/content-page.service.ts]: Yes, i removed the commented code. The correct function to use is: fetchContentPages

**Review by BavoVdg — APPROVED** [review]

**Pull request merged**